stream-chat-android-ui-utils/src/main/kotlin/io/getstream/chat/android/uiutils/extension/Context.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2014-2022 Stream.io Inc. All rights reserved.
+ *
+ * Licensed under the Stream License;
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://github.com/GetStream/stream-chat-android/blob/main/LICENSE
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.getstream.chat.android.uiutils.extension
+
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import android.provider.Settings
+import android.view.LayoutInflater
+
+/**
+ * Creates a new [LayoutInflater] instance associated with a particular Context. Ths context
+ * supplies the theme from which the default values for their attributes are retrieved.
+ */
+public inline val Context.inflater: LayoutInflater
```

Comment:
There seem to be no usages of this in code.
Do we need it, and more importantly, do we need it to be publicly exposed?